qemu/tests/cfg/numa_opts.cfg

```diff
@@ -92,3 +93,19 @@
               extra_params += " -numa node,mem=2560,cpus=5,nodeid=2"
               extra_params += " -numa node,mem=512,cpus=3-4,nodeid=0"
               extra_params += " -numa node,mem=1024,cpus=0-2,nodeid=1"
+    - nodes.128:
+        type = numa_maxnodes
+        numa_nodes = 128
+        mem_fixed = 4G
+        use_mem = no
+        node_size = 32M
```

Comment:
In my opinion, better add  `vm_mem_minimum=4G`  for x86 (32M*128) although low chance to get this kind of machine. what do you think ?

Reply:
@yanan-fu I've thought about it. Considering we normally run memory test on host with more than 8G memory and never hit the issue, I didn't add the param for x86. But I guess it should be better to treat x86 and ppc equally, will update, thanks.